src/main/kotlin/com/exactpro/th2/codec/api/IPipelineCodecFactory.kt

```diff
@@ -22,8 +22,9 @@ import java.io.InputStream
 interface IPipelineCodecFactory : AutoCloseable {
     val protocols: Set<String>
         get() = setOf(protocol)
-    @Deprecated("Please migrate to the protocols property")
+    @Deprecated("Please migrate to the protocols property", replaceWith = ReplaceWith("protocols"))
     val protocol: String
+        get() = error("This property deprecated and should not be used")
```

Comment:
I think we have deprecated it for enough time. Let's just remove this prop from the interface